lib/rspec-puppet-facts.rb

```diff
@@ -161,6 +178,13 @@ def self.spec_facts_os_filter
     ENV['SPEC_FACTS_OS']
   end
 
+  # If SPEC_FACTS_STRICT is set to `yes`, RspecPuppetFacts will error on missing FacterDB entries, instead of warning & skipping.
+  # @return [nil,String]
+  # @api private
+  def self.spec_facts_strict?
+    ENV['SPEC_FACTS_STRICT'] == 'yes'
```

Comment:
Rather than needing specifically 'yes', could this be set to anything (or set to anything that isn't 'no')?